src/integration/integration_suite_test.go

```diff
@@ -391,15 +398,27 @@ func clearDatabase() {
 	Expect(err).NotTo(HaveOccurred())
 }
 
-func insertPolicy(appId string, scalingPolicy models.ScalingPolicy) {
+// func insertPolicy-ss(appId string, scalingPolicy models.ScalingPolicy) {
```

Comment:
@qibobo  Just wanted to understand the reason behind keeping this piece of commented code